crates/configuration/src/network.rs

```diff
@@ -324,6 +325,42 @@ impl NetworkConfigBuilder<Initial> {
         Self::default()
     }
 
+    /// uses the default options for both the relay chain and the nodes
+    /// the only required fields are the name of the nodes,
+    /// and the name of the relay chain ("rococo-local", "polkadot", etc.)
+    pub fn with_nodes_and_chain(
+        node_names: Vec<String>,
+        relay_name: &str,
```

Comment:
Just `naming` but I think is better `with_chain_and_nodes` and also change the order of the args.